packages/frontend/src/contexts/Authentication.tsx

```diff
@@ -38,6 +39,11 @@ export const AuthenticationProvider = ({
     refetchQueries: [GET_CURRENT_USER],
     awaitRefetchQueries: true,
   })
+  useEffect(() => {
+    if (currentUser) {
+      datadogRum.setUser(currentUser)
```

Comment:
comment: Actually i think we might have been above to combine this into the `onCompleted` callback earlier, but i am removing that callback in #312, so just a comment